pom.xml

```diff
@@ -43,8 +43,8 @@
 
 
     <properties>
-        <maven.compiler.source>1.6</maven.compiler.source>
-        <maven.compiler.target>1.6</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Is there a reason why you have upgrade the required version of Java to Java 8?  There are many users of MockServer is large organisation with out of date versions of Java.  If you can update the PR to remove this and ensure the tests are passing then it can be merged into master.